website/pages/docs/plugins/sources/k8s/overview.mdx

```diff
@@ -47,5 +51,21 @@ rules:
   - get
   - list
 EOF
-$ kubectl create clusterrolebinding cloudquery-cluster-read-$service_account-binding --clusterrole=cloudquery-cluster-read --serviceaccount=$service_account
+```
+
+Then, create a cluster role binding for that role and the service account the `cloudquery` pod uses:
```

Comment:
Next, the cluster role and service account will need to be linked via a cluster role binding.

The following creates a cluster role binding for the role we created above and the service account for the `cloudquery` pod.